assets/javascripts/modules/validatorFocus.js

```diff
@@ -7,10 +7,26 @@
     }
 
     $('.error-summary a').on('click', function(e) {
+      var $this = $(this);
+      var focusId = $this.attr('data-focuses');
+      if (!focusId)
+      {
+        focusId = ("" + $this.attr("href")).trim();
+        if (focusId.indexOf("#") !== 0)
+          return;
+      }
+      else
+      {
+        focusId = "#" + focusId;
+      }
+
+      var inputToFocus = $(focusId);
+      if (!inputToFocus.size())
```

Comment:
`.size()` is deprecated as of `1.8` would prefer `length` here https://api.jquery.com/size/